docs/user-guide/helm.md

```diff
@@ -170,18 +170,18 @@ Values injections have the following order of precedence
     highest -> parameters
 ```
 
-so values/valuesObject trumps valueFiles, and parameters trump both.
+So valuesObject trumps values, and both valuesObject and values trump valueFiles.
```

Comment:
Could you make it a bit clearer that when valuesObject is used, values is just ignored ?
For the others, the yaml are merged but not this one

Reply:
updated.